assume/world.py

```diff
@@ -116,18 +116,33 @@ def __init__(
             self.db_uri = make_url(database_uri)
             db = create_engine(self.db_uri)
             connected = False
-            while not connected:
+            attempts = 0
+            max_attempts = 5
+
+            while not connected and attempts < max_attempts:
                 try:
                     with db.connect():
                         connected = True
-                        logger.info("connected to db")
+                        logger.info("Connected to the database")
                 except OperationalError as e:
-                    logger.error("could not connect to %s, trying again", database_uri)
+                    attempts += 1
+                    logger.error(
+                        "Could not connect to %s, trying again (%d/%d)",
+                        database_uri,
+                        attempts,
+                        max_attempts,
+                    )
                     # log error if not connection refused
                     if not e.code == "e3q8":
                         logger.error("%s", e)
                     time.sleep(2)
```

Comment:
```suggestion
                    time.sleep(2**attempts)
```
This increases total wait time to 1 minute in total which should be more suitable in some scenarios
(2 + 4 + 8 + 16 + 32)